src/libsyntax/ext/tt/quoted.rs

```diff
@@ -50,11 +50,17 @@ pub struct SequenceRepetition {
     /// The optional separator
     pub separator: Option<Token>,
     /// Whether the sequence can be repeated zero (*), or one or more times (+)
-    pub op: KleeneOp,
+    pub op: KleeneToken,
     /// The number of `Match`s that appear in the sequence (and subsequences)
     pub num_captures: usize,
 }
 
+#[derive(Clone, PartialEq, RustcEncodable, RustcDecodable, Debug, Copy)]
+pub struct KleeneToken {
+    pub span: Span,
+    pub op: KleeneOp,
+}
```

Comment:
Could you add a constructor function `fn new(op: KleeneOp, span: Span) -> KleeneToken { ... }` and use it instead of `KleeneToken { ... }`.